File: app/src/main/java/com/twt/zhihu/zhihudaily/Model/MainBean.java
package com.twt.zhihu.zhihudaily.Model;

import java.util.List;

public class MainBean {
    public String date;
    public List<StoriesBean> stories;
    public List<TopStoriesBean> top_stories;


    public static class StoriesBean {
        public String date;
        public int type;
        public int id;
        public String ga_prefix;
        public String title;
        public boolean multipic;
        public List<String> images;
    }


    public static class TopStoriesBean {
        public String image;
        public int type;
        public int id;
        public String ga_prefix;
        public String title;
    }
}

File: app/src/main/java/com/twt/zhihu/zhihudaily/model/MainBean.kt
package com.twt.zhihu.zhihudaily.model

class MainBean {
    var date: String? = null
    lateinit var stories: List<StoriesBean>
    var top_stories: List<TopStoriesBean>? = null

    class StoriesBean {
        var date: String? = null
        var type: Int = 0
        var id: Int = 0
        var ga_prefix: String? = null
        var title: String? = null
        var multipic: Boolean = false
        var images: List<String>? = null
    }


    class TopStoriesBean {
        var image: String? = null
        var type: Int = 0
        var id: Int = 0
        var ga_prefix: String? = null
        var title: String? = null
    }
}
